docs/managed-victoriametrics/overview.md

```diff
@@ -39,4 +38,4 @@ Managed VictoriaMetrics comes with the following features:
 * Every Managed VictoriaMetrics instance runs in an isolated environment, so instances cannot interfere with each other;
 * Managed VictoriaMetrics instance can be scaled up or scaled down in a few clicks;
 * Automated backups;
-* Pay only for the actually used compute resources - instance type, disk and network.
+* Pay only for the actually used resources - instance type, disk and network.
```

Comment:
should we use the same measurements as in billing?
compute, storage, traffic?

Reply:
I used the same measurements. Users will better understand our information in the site